opentelemetry-instrumentation/README.rst

```diff
@@ -16,30 +16,92 @@ Installation
 
 This package provides a couple of commands that help automatically instruments a program:
 
+
+opentelemetry-bootstrap
+-----------------------
+
+::
+
+    opentelemetry-bootstrap --action=install|requirements
+
+This commands inspects the active Python site-packages and figures out which
+instrumentation packages the user might want to install. By default it prints out
+a list of the suggested instrumentation packages which can be added to a requirements.txt
+file. It also supports installing the suggested packages when run with :code:`--action=install`
+flag.
+
+The command also installs the OTLP exporter by default. This can be overriden by specifying another
+exporter using the `--exporter` or `-e` CLI flag. Multiple exporters can be installed by specifying
+the flag more than once. Run `opentelemetry-bootstrap --help` to list down all supported exporters.
+
+Manually specifying exporters to install:
+
+::
+
+    opentelemetry-bootstrap -e=otlp -e=zipkin
+
+
 opentelemetry-instrument
 ------------------------
 
 ::
 
     opentelemetry-instrument python program.py
 
+The instrument command will try to automatically detect packages used by your python program
+and when possible, apply automatic tracing instrumentation on them. This means your program
+will get automatic distrubuted tracing for free without having to make any code changes
+at all. This will also configure a global tracer and tracing exporter without you having to
+make any code changes. By default, the instrument command will use the OTLP exporter but
+this can be overrided when needed.
+
+The command supports the following configuration options as CLI arguments and environments vars:
+
+
+* ``--trace-exporter`` or ``OTEL_TRACE_EXPORTER``
```

Comment:
Java's using `OTEL_EXPORTER` and we have a spec pending for that

https://github.com/open-telemetry/opentelemetry-specification/issues/710

We plan on updating the Java usage to accept a list to solve the issue of trace vs metrics.

Do you think you can use that name here?

/cc @trask